mlir/lib/Conversion/TosaToLinalg/TosaToLinalg.cpp

```diff
@@ -506,6 +506,13 @@ createLinalgBodyCalculationForElementwiseOp(Operation *op, ValueRange args,
     }
   }
 
+  // tosa::CustomOp
+  if (auto customOp = dyn_cast<tosa::CustomOp>(op)) {
+    return llvm::StringSwitch<Value>(customOp.getIdentifierAttr().str())
+        .Case("atan2", rewriter.create<math::Atan2Op>(loc, resultTypes, args))
```

Comment:
In llvm-project, isn't it usual to have a `FileCheck` test for everything?

Reply:
Agreed! Should we revert the merge and add a test?

Reply:
A new PR should be enough